rti/mosaic-starter/src/main/java/org/eclipse/mosaic/starter/MosaicStarter.java

```diff
@@ -230,10 +231,9 @@ private Logger getLogger() {
 
     protected CRuntime loadRuntimeConfiguration(Path configurationPath) throws ExecutionException {
         try {
-            try (InputStream resource = loadResource(configurationPath)) {
-                return new XmlMapper()
-                        .disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES)
-                        .readValue(resource, CRuntime.class);
+            try (InputStream inputStream = loadResource(configurationPath)) {
+                InputStreamReader reader = new InputStreamReader(inputStream);
+                return new Gson().fromJson(reader, CRuntime.class);
```

Comment:
It would be better to use the predefined method `loadJsonConfiguration` instead:
```java
return loadJsonConfiguration(configurationPath, CRuntime.class);
```